app/uk/gov/hmrc/accessibilitystatementfrontend/views/fragments/mobile/HowAppWasTested.scala.html

```diff
@@ -33,10 +33,10 @@ <h2 class="govuk-heading-l">@{messages("testing.heading", messages("general.app"
         @if(displayAutomatedTestingOnlyContent) {
             <p class="govuk-body">@{messages("testing.last-tested.automated", utils.Dates.formatDate(dateToLocalDate(serviceLastTestedDate.get)), messages("general.app"))}</p>
         } else {
-            <p class="govuk-body">@{messages("testing.last-tested", utils.Dates.formatDate(dateToLocalDate(serviceLastTestedDate.get)), messages("general.app"))}</p>
+            <p class="govuk-body">@{messages("testing.last-tested.app", utils.Dates.formatDate(dateToLocalDate(serviceLastTestedDate.get)))}</p>
```

Comment:
i do prefer this approach, but it kind of introduces some inconsistency/redundancy, where the English non-app message still interpolates the word "service", but the Welsh version is hardcoded in messages.cy

Reply:
@ellamdav It's a tricky one - the problem is in some cases the prefix is different for "gwasanaeth" (service), or "ap" (app). For example, it's "y gwasanaeth" but "yr ap", so we can't just drop in the word "gwasanaeth" or "ap" using parameters.

Now that we have the different fragments, I'm tempted to re-write the whole thing to get rid of parameterised messages - what do you think?